client/components/marketing-survey/cancel-jetpack-form/jetpack-cancellation-survey.tsx

```diff
@@ -45,6 +45,10 @@ export default function JetpackCancellationSurvey( {
 			id: 'dont-need-website',
 			answerText: translate( 'I no longer need a website.' ),
 		},
+		{
+			id: 'could-not-get-support',
```

Comment:
We also need to add this `id` to fbhepr%2Skers%2Szp%2Sznexrgvat%2Sfheirl%2Sshapgvbaf.cuc%3Se%3Qp9oo77p5%23184-og so it shows up in MC.

Reply:
Just saw the other task, disregard this comment 😆